### ISSUE TITLE
Deleting an Engagement Plan then updating one of its tasks causes null pointer exception

### ISSUE THREAD

**Issue by njjc** [SalesforceFoundation/Cumulus#2372]
Create an Engagement Plan so that Tasks are applied
Delete Engagement Plan
Edit the Status of one of the newly created Tasks
```
System.NullPointerException: Attempt to de-reference a null object: Class.nictest.EP_TaskDependency_TDTM.updateDependentTasks: line 99, column 1
```

Tasks without both EP Task and EP lookups should behave like normal tasks without any Engagement Plan automation. No exception should be thrown.

**Comment by njjc** [comment]
**lurch: add

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x80000004DBLkAAO" target="_blank">W-016935</a>

**Comment by jlantz** [comment]
Included in beta release 3.85 (Beta 8)
